service/dataset.py

```diff
@@ -23,17 +23,11 @@
                         },
                         "budget":3.0}}
 
+RELEASED_DATASETS = []
 
 KNOWN_DATASET_TYPE_KEYS = ["csv_details", "dataverse_details"]
 
-def read(dataset_request):
-    """Get information needed to load the dataset
-
-    :param info: The dataset to read and budget to use.
-    :type info: dict {"dataset_name": str, "budget":int}
-    :return: A dataset document that contains the type and info of the dataset
-    :rtype: dict{"dataset_type": str, dataset_key: dict}
-    """
+def _read_helper(dataset_request):
```

Comment:
This should take the dict to read from as input and be wrapped by _read_released, _read.

Pretty sure we are always reading from the private datasets now